libraries/APM_Control/AR_AttitudeControl.cpp

```diff
@@ -68,18 +68,24 @@ const AP_Param::GroupInfo AR_AttitudeControl::var_info[] = {
     // @Param: _STR_RAT_FLTT
     // @DisplayName: Steering control Target filter frequency in Hz
     // @Description: Target filter frequency in Hz
+    // @Range: 0.000 100.000
+    // @Increment: 0.1
```

Comment:
The number of decimal places is not correct.
I would suggest matching the number of digits.
The same goes for the other items.

Reply:
This would need to be also fixed in master, before I could backport. That would be another PR